### DISCUSSION TITLE
Search result items should include folder (path)

### DISCUSSION THREAD

**Discussion by voodoopt** [dani-garcia/vaultwarden#3524] [Ideas]
Has anyone ever needed this feature?
I have the similar problem and keePassXC has this and it's very handy
neither on the web nor in the linux app is this feature available

https://community.bitwarden.com/t/search-result-items-should-include-folder-path/1287/1
![Screenshot_2023-05-12_16-06-41](https://github.com/dani-garcia/vaultwarden/assets/5041640/874df5ef-c4b4-45f7-a4e4-14b3bc826b33)

**Comment by BlackDex** [comment]
While this might be interesting in general. Why post this here at Vaultwarden?

**Reply by voodoopt** [reply]
this option makes perfect sense in Vaultwaden because anyone with many organizations can easily do a global search and get a more objective result
by the development of the bitwarden page it seems to me that developing something similar is always for the paid version because it is the only one that has organizations
And the free versions are left out
It will be a good suggestion for vaultwarden as we have the possibility of multiple organizations.

**Reply by BlackDex** [reply]
The free version also has two free organizations as far as i know.
But besides that. We do not do develop on the web-vault it self. We also don't want to change to much on the web-vault it self in general.

We do sometimes provide patches for the web-vault if there is something wrong, but that is upstream.

So, for any new features, this needs to be done at Bitwarden. Also, it probably is good to have this available for all clients, instead of only the web-vault supplied by Vaultwarden, like the Desktop app for example.
